01_prepare_host.sh

```diff
@@ -38,20 +38,36 @@ if ! which kustomize 2>/dev/null ; then
     sudo mv kustomize /usr/local/bin/.
 fi
 
-# Download Ironic binary and Ironic inspector image
+# Clean-up any old ironic containers
+for name in ironic ironic-inspector dnsmasq httpd mariadb ipa-downloader; do
+    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill $name
+    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" rm $name -f
+done
 
-mkdir -p "$IRONIC_DATA_DIR/html/images"
-pushd "$IRONIC_DATA_DIR/html/images"
-if [ ! -f ironic-python-agent.initramfs ]; then
-    curl --insecure --compressed -L https://images.rdoproject.org/master/rdo_trunk/current-tripleo-rdo/ironic-python-agent.tar | tar -xf -
+if [[ "${CONTAINER_RUNTIME}" == "podman" ]]; then
+  # Remove existing pod
+  if  sudo "${CONTAINER_RUNTIME}" pod exists ironic-pod ; then
+      sudo "${CONTAINER_RUNTIME}" pod rm ironic-pod -f
+  fi
+  # Create pod
+  sudo "${CONTAINER_RUNTIME}" pod create -n ironic-pod
 fi
 
-for IMAGE_VAR in IRONIC_IMAGE IRONIC_INSPECTOR_IMAGE ; do
+# Pull images we'll need
+for IMAGE_VAR in IPA_DOWNLOADER_IMAGE VBMC_IMAGE SUSHY_TOOLS_IMAGE; do
     IMAGE=${!IMAGE_VAR}
     sudo "${CONTAINER_RUNTIME}" pull "$IMAGE"
 done
 
-## Download centos 7 qcow2 image
+# Download IPA and CentOS 7 Images
+mkdir -p "$IRONIC_DATA_DIR/html/images"
```

Comment:
I think the path can be change to a vaiable since it's reused a couple of times ?